skglm/solvers/fista.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from numba import njit
+from skglm.solvers.base import BaseSolver
+
+
+@njit
+def _prox_vec(w, z, penalty, lipschitz):
```

Comment:
I like it for now. I think it's painful for the user to define both prox and prox1d

Reply:
+1